src/electionguard_cli/cli_steps/input_retrieval_step_base.py

```diff
@@ -31,3 +32,12 @@ def get_first_value(text: Optional[InternationalizedText]) -> str:
         self.print_value("Candidates", len(manifest.candidates))
         self.print_value("Contests", len(manifest.contests))
         self.print_value("Ballot Styles", len(manifest.ballot_styles))
+
+    @staticmethod
+    def _get_guardians(number_of_guardians: int, quorum: int) -> List[Guardian]:
```

Comment:
This was just in e2e, but now setup uses it too, so I promoted it to the base class

Reply:
This should exist in a helper in the `electionguard_tools` if you want to really promote it. 
This line is already used in there to create the gaurdians in the KeyCeremonyOrchestrator. 

Would reduce repetition overall.

Reply:
See this already existing static method.
https://github.com/microsoft/electionguard-python/blob/f0d6f01400d2e71feb7cc1e51af64aa202fa9097/src/electionguard_tools/helpers/key_ceremony_orchestrator.py#L11